dispatcher.go

```diff
@@ -60,6 +98,11 @@ func dispatchAddServer(serviceName string) {
 	// connect to server
 	// there can be more than 1 message outstanding toards same server
 
+	if drsmInitDone == false {
+		Drsm, _ = InitDrsm()
```

Comment:
Why not checking if result (`err`) is not `nil` and then making `drsmInitDone = true`